public/src/js/widgets/columns/packages.js

```diff
@@ -15,7 +16,6 @@ export default class Package extends ColumnWidget {
     constructor(params, element) {
         super(params, element);
         this.allPackages;
-        this.searchedPackages = ko.observableArray();
```

Comment:
I see that it's duplicated further down, are we sure it should be an `observable` and not an `observableArray`?